ui/css/treeherder-bugfiler.css

```diff
@@ -5,6 +5,23 @@
     padding: 20px;
 }
 
+#modalProductFinderButton {
+    margin: -4px 0 0 5px;
+    line-height: 16px;
+}
+
+#suggestedProducts {
```

Comment:
These margin attributes seem quite precise, but I wonder if you couldn't use some of the bootstrap utility spacing classes for some or if it'd be worth making them into general spacing classes to reuse?

Reply:
Yeah, this is a really good point.  I spent some time laying this out with just the bootstrap classes.  Very educational.  :)  But I eliminated most of the CSS file in the process.